src/components/ManageUsers/manage-users.css

```diff
@@ -1,58 +1,50 @@
 .usersearch__h1 {
+  margin-left: 4px;
   font-size: 16px;
   color: #e3893e;
   display: right;
   font-weight: 200px;
-  margin-right: 10px;
-  text-align: right;
 }
 
 .usersearch__container {
-  padding: 16px;
   min-height: 10vh;
   background: #000000;
   background-size: cover;
+  margin-right: auto;
   display: flex;
   flex-direction: column;
-  margin-left: auto;
 }
 
 .usersearch__search {
   margin-top: 10px;
   margin-bottom: 10px;
-  gap: 10px;
   width: 100%;
-  max-width: 300px;
-  margin-right: 10px;
-  margin-left: auto;
+  max-width: 500px;
 }
 
 .usersearch__input {
   height: 16px;
   width: 100%;
-  max-width: 300px;
```

Comment:
why did you add 500px width to the container (usersearch__search) and here 600? It's enough to put it in one place